api/src/pcapi/core/finance/models.py

```diff
@@ -238,6 +238,7 @@ class BankAccount(PcObject, Base, Model, DeactivableMixin):
         foreign_keys="BankAccountStatusHistory.bankAccountId",
         uselist=True,
     )
+    lastCegidSync = sqla.Column(sqla.DateTime, nullable=True)
```

Comment:
```suggestion
    lastCegidSyncDate = sqla.Column(sqla.DateTime, nullable=True)
```
Comme ça on sait tout de suite si c'est une date (et pas un booléen)